woocommerce/woocommerce-ios
# Refunds - Yosemite layer

Closes #1095 

This PR ties together all the changes to all the layers. This PR should build, run, and pass all unit tests.

## Changes
- Note: the `price` in `OrderItem` and `OrderItemRefund` is set to the `NSDecimalNumber` type because not all of the `NSDecimalNumber` APIs have been exposed and converted for use with `Decimal`. `NSDecimalNumber` can detect string to number conversions and whether the string became a valid number (or not).
- `OrderItemRefund` CodingKeys updated: `refundTotal = refund_total` was incorrect. The actual field is `total`.
- Add Yosemite's `ReadOnlyConvertible` extensions to `OrderItemTax`, `OrderItemRefund`, `OrderItemTaxRefund`, and `Refund`.

### To test
- check out and build develop on your sim
- check out and build this branch
- make sure running this build doesn't break (and migration happens smoothly)
- run all unit tests

Update release notes:
